client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -62,77 +63,17 @@ export class GoogleMyBusiness extends SharingService {
 		}
 	}
 
-	/*
-	 * We render a custom logo here because Google My Business is not part of SocialLogos
-	 */
 	renderLogo() {
+		// Render a custom logo here because Google My Business is not part of SocialLogos
 		return (
-			/* eslint-disable wpcalypso/jsx-classname-namespace */
-			/* eslint-disable max-len */
-			<svg
+			<img
+				alt={ translate( 'Google My Business logo' ) }
 				className="sharing-service__logo"
-				style={ { padding: 6 + 'px' } }
-				version="1.1"
 				height="36"
+				src={ '/calypso/images/google-my-business/logo.svg' }
```

Comment:
Didn't we agree this "icon" should stay inline in the code otherwise it displays with delay?
https://github.com/Automattic/wp-calypso/pull/23560/files#r177366538

I'm fine making it a new component, but I'm not convinced on the external svg approach.

Reply:
Ah yes. I'll change that. Where do you think we should create that component?

Reply:
we could have a simple `google-my-business/logo.js` component and I think it would be fine for now.